web_src/css/shared/flex-list.css

```diff
@@ -20,6 +20,7 @@
   flex-grow: 1;
   flex-basis: 60%; /* avoid wrapping the "flex-item-trailing" too aggressively */
   min-width: 0; /* make the "text truncate" work, otherwise the flex axis is not limited and the text just overflows */
+  gap: .25rem;
```

Comment:
By the way, if we'd like to add the gap here (a framework level change), then we need to check other places and remove unnecessary patches like below, otherwise the space doubles.

![image](https://github.com/user-attachments/assets/25419a7b-1d08-4a01-a1db-563cb5778692)

Reply:
-> Refactor some trivial problems#34959